File: javaspring/src/main/java/patterns/sample/utils/Pagination/Transaction.java
package patterns.sample.utils.Pagination;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Transaction {
    @Id
    private String id;

    private LocalDateTime time;

    private BigDecimal amount;

    public String getId() {
        return id;
    }

    public void setId() {
        this.id = UUID.randomUUID().toString();
    }

    public LocalDateTime getTime() {
        return time;
    }

    public BigDecimal getAmount() {
        return amount;
    }

    public void setAmount(BigDecimal amount) {
        this.amount = amount;
    }

    public void setTime(LocalDateTime time) {
        this.time = time;
    }
}

File: dotnet/Utils/PaginationForList/Transaction.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace sampleApi.Utils.PaginationForList
{
    public class Transaction
    {
        [Key]
        public string id { get; set; }
        public DateTime time { get; set; }
        public Decimal amount { get; set; }
    }
}